batchflow/models/eager_torch/base.py

```diff
@@ -836,8 +848,8 @@ def train(self, *args, feed_dict=None, fetches=None, use_lock=False, train_mode=
         microbatch : int, bool or None
             If int, then size of chunks to split every batch into. Allows to process given data sequentially,
             accumulating gradients from microbatches and applying them once in the end.
-            If True, then value from config is used (default value is not to use microbatching).
-            If False or None, then microbatching is not used.
+            If True or None, then value from config is used (default value is not to use microbatching).
```

Comment:
What about making `True` default value here and making `None` behave like `False`? Then if user doesn't pass anything - we're using value from config, if user passes something that makes `not microbatching == True`, including `mictobatching==0`, then we don't use microbatching. This will be more consistent with meaning of microbatching key in config

Reply:
Done